src/LoadOutWeapon.js

```diff
@@ -0,0 +1,27 @@
+import ItemConfig from './ItemConfig.js';
+import Polarity from './Polarity.js';
+import { mapColors, parseDate } from './utils.js';
+
+export default class LoadOutWeapon {
+  constructor(weapon) {
+    this.itemId = weapon.ItemId.$oid;
+    this.uniqueName = weapon.ItemType;
+    this.name = this.ItemName;
+    this.configs = weapon.Configs.map((c) => new ItemConfig(c));
+    this.upgradeType = weapon.UpgradeType;
+    this.upgradeFingerprint = weapon.UpgradeFingerprint;
+    this.features = weapon.Features;
+    this.upgradeVer = weapon.UpgradeVer;
+    this.xp = weapon.XP;
+    this.polarized = weapon.Polarized;
```

Comment:
Are these false or undefined when not polarized? If false, 👍, otherwise we should `?? false` Boolean fields

Reply:
```js
/**
 * How many times a forma was applied to the weapon
 * @type {number}
 */
this.polarized = weapon.Polarized;
```

Reply:
currently writing out some docs lol